firmware/src/pdu.c

```diff
@@ -68,13 +68,36 @@ main(void)
 	/* ============================================================== */
 
 
+	int counter_above50[6] = {0,0,0,0,0,0};
+	int counter_above30[6] = {0,0,0,0,0,0};
+
 	for (;;) {
 		pdu_adc_run();
 
+		// Compare the channel to the fuse
 		if (pdu_fuse_trigger) {
 			// FUSE LOGIC HERE!
 			for (i = 0; i == 6; i++) {
-				// if (pdu_channel_currents[i] > pdu_fuse_currents[i]) {
+				if (pdu_channel_currents[i] > pdu_fuse_currents[i]) {
```

Comment:
We don't have a `pdu_fuse_currents` any more since we're using the `pdu_channels` structs.